operator/pkg/ciliumendpointbatch/reconciler_test.go

```diff
@@ -0,0 +1,212 @@
+			Name: "Ceb-apple-one",
+			Annotations: map[string]string{
+				annotation.CiliumEndpointBatchQueueInfo: CEBatchingModeFcfs,
+			},
+		},
+		Endpoints: []capi_v2a1.CoreCiliumEndpoint{
+			{
+				Name:       "cilium-abcd-123",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+			{
+				Name:       "cilium-abcd-456",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+		},
+	}
+
+	// Test CEB object, with 2 CEPs packed in it.
+	Ceb2 = &capi_v2a1.CiliumEndpointBatch{
+		TypeMeta: meta_v1.TypeMeta{
+			Kind:       "CiliumEndpointBatch",
+			APIVersion: capi_v2a1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name: "Ceb-apple-two",
+			Annotations: map[string]string{
+				annotation.CiliumEndpointBatchQueueInfo: CEBatchingModeFcfs,
+			},
+		},
+		Endpoints: []capi_v2a1.CoreCiliumEndpoint{
+			{
+				Name:       "cilium-nfgt-123",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+			{
+				Name:       "cilium-nfgt-456",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+		},
+	}
+
+	// Test CEB object, with 2 CEPs packed in it.
+	Ceb3 = &capi_v2a1.CiliumEndpointBatch{
+		TypeMeta: meta_v1.TypeMeta{
+			Kind:       "CiliumEndpointBatch",
+			APIVersion: capi_v2a1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name: "Ceb-apple-three",
+			Annotations: map[string]string{
+				annotation.CiliumEndpointBatchQueueInfo: CEBatchingModeFcfs,
+			},
+		},
+		Endpoints: []capi_v2a1.CoreCiliumEndpoint{
+			{
+				Name:       "cilium-tkld-123",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+			{
+				Name:       "cilium-tkld-456",
+				Namespace:  "kube-system",
+				IdentityID: 364748,
+			},
+		},
+	}
+)
+
+// Create a fake cilium Client and add prepend reactor for Create/Update/Delete
+func ciliumClient() clientset.CiliumV2alpha1Interface {
+	client := fake.NewSimpleClientset()
+	client.PrependReactor("create", "ciliumendpointbatches", k8stesting.ReactionFunc(func(action k8stesting.Action) (bool, runtime.Object, error) {
+		ciliumEndpointBatch := action.(k8stesting.CreateAction).GetObject().(*capi_v2a1.CiliumEndpointBatch)
+		ciliumEndpointBatch.UID = CEBReconcilerUID
+		return false, ciliumEndpointBatch, nil
+	}))
+	client.PrependReactor("update", "ciliumendpointbatches", k8stesting.ReactionFunc(func(action k8stesting.Action) (bool, runtime.Object, error) {
+		ciliumEndpointBatch := action.(k8stesting.UpdateAction).GetObject().(*capi_v2a1.CiliumEndpointBatch)
+		ciliumEndpointBatch.Generation = CEBReconcilerGenerate
+		return false, ciliumEndpointBatch, nil
+	}))
+	client.PrependReactor("delete", "ciliumendpointbatches", k8stesting.ReactionFunc(func(action k8stesting.Action) (bool, runtime.Object, error) {
+		return false, nil, nil
+	}))
+	return client.CiliumV2alpha1()
+}
+
+// Test Reconciler by creating CEBs, Updating CEBs and Deleting CEBs
+func TestCiliumReconcile(t *testing.T) {
+	var e []string
```

Comment:
call this "empty", and make it a const.

Reply:
I'll update variable name as `empty`. Not sure how do we declare `empty string` as constant. https://blog.golang.org/constants